src/plugin/server.js

```diff
@@ -31,12 +33,15 @@ if (process.env.NODE_ENV !== "jarvis_dev") {
     res.sendFile(path.join(__dirname + "../../dist/bin/index.html"))
   );
 } else {
-  app.get("/", (_, res) => res.send("Jarvis client is running on: 1337"));
+  app.get("/", (_, res) => res.send(`Jarvis client is running on: ${PORT}`));
 }
```

Comment:
The client app uses Socket IO and we probably need to tell it the port isn't 1337 anymore. Is that correct? I hope we don't need to really specify anything on client btw

https://github.com/zouhir/jarvis/blob/master/src/client/components/board.js#L13

Reply:
fixed this by adding document.location.port to the socket IO conbection clientside